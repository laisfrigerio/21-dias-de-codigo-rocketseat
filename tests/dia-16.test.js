const { JSDOM } = require('jsdom')
const fs = require('fs')
const path = require('path')

const html = fs.readFileSync(path.resolve(__dirname, '../dia-16/index.html'), 'utf8')

let dom
let container

describe('dia 16', () => {
  beforeEach(() => {
    dom = new JSDOM(html, { runScripts: 'dangerously' })
    container = dom.window.document.body
  })

  test('should match the consume API challenge', () => {
    expect(container).toMatchSnapshot();
  });
})