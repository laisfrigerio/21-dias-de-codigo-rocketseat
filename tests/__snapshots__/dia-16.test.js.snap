// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dia 16 should match the consume API challenge 1`] = `
<body
  class="main-container no-toggle"
  id="particles-js"
>
  
  
  <header />
  
  
  <article
    class="card-quiz"
  >
    
    
    <div
      class="header"
    >
      
      
      <h1
        tabindex="0"
      >
        Sports
      </h1>
      
      
      <span
        class="score"
        tabindex="0"
      >
        0
      </span>
      
    
    </div>
    
    
    <form
      action="#"
    >
      
      
      <h2
        class="quiz-question"
        tabindex="0"
      />
      
      
      <ul
        class="quiz-answers"
        role="list"
      >
        
      
      </ul>
      
    
    </form>
    
    
    <div
      class="footer"
    >
      
      
      <button
        class="verify"
        role="button"
        tabindex="0"
        type="button"
      >
        VERIFICAR
      </button>
      
    
    </div>
    
    
    <div
      class="quiz-end"
    />
    
    
    <div
      class="spinner"
    />
    
  
  </article>
  

  
  <script
    src="https://cdn.jsdelivr.net/npm/particles.js@2.0.0/particles.min.js"
  />
  
  
  <script
    src="../assets/js/theme.js"
  />
  
  
  <script
    src="../assets/js/particles.js"
  />
  
  
  <script
    src="../assets/js/include-html.js"
  />
  
  
  <script
    src="./load-questions.js"
  />
  
  
  <script
    src="../assets/js/quiz.js"
  />
  
  
  <script
    src="./scripts.js"
  />
  



</body>
`;
